### ISSUE TITLE
I get a TypeErrorCannot read property 'request' of undefined error and cannot upload.

### ISSUE THREAD

**Issue by secmin** [opencoconut/coconutjs#9]
I get the following error.
Is there a solution?

```
TypeErrorCannot read property 'request' of undefined
at null.null (/opt/ee/node_modules/coconutjs/job.js:15:28)
at null.Promise (null:null:null)
at Function.create (/opt/ee/node_modules/coconutjs/job.js:14:14)
at Object.module.exports (/steps/nodejs.js:23:34)
at MessagePort.messageHandler (/opt/nodejs/node_modules/@pipedreamhq/execution-environment/launch_worker.js:427:41)
at MessagePort.[nodejs.internal.kHybridDispatch] (internal/event_target.js:354:41)
at MessagePort.exports.emitMessage (internal/per_context/messageport.js:18:26)
```

**Comment by thehypeapp** [comment]
any update on this? also running into this issue, can't create jobs

**Comment by brunoceleste** [comment]
We just fixed the issue, there was a bug when using Promise. New version available 3.0.2
Thanks!
